test/Feature/DeleteSpec.hs

```diff
@@ -57,14 +57,6 @@ spec =
           , matchHeaders = ["Content-Range" <:> "*/*"]
           }
 
-      it "actually clears items ouf the db" $ do
```

Comment:
This test is obsolete, because "persistance" of all kinds of requests is now extensively covered in `RollbackSpec`.